packages/vite-plugin-cloudflare/src/index.ts

```diff
@@ -393,6 +401,11 @@ export function cloudflare(pluginConfig: PluginConfig = {}): vite.Plugin[] {
 
 							await sendResponse(res, response as any);
 						} catch (error) {
+							// Do not bubble client abort errors up to the Vite server
+							if (error instanceof Error && error.name === "AbortError") {
+								return;
+							}
+
```

Comment:
NIT: since we do the same in another catch block how about we move this to a helper function:

```ts
} catch (error) {
  if (!isAbortError(error)) {
    next(error);
  }
}
```